### ISSUE TITLE
site.yml: url: https://github.com/owncloud/docs.git eliminates local html content generation

### ISSUE THREAD

**Issue by mmattel** [owncloud/docs#486]
## What Type Of Content Change Is This?

- [ ] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [x] Bug Fix to Existing Content

## Which Manual Does This Relate To?

- [ ] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [x] General

## What Needs to be Documented?

Seems that adding docs.git as source in `site.yml`, eliminates local content html generation (`make html`), as it take the content from the repository.
```
content:
  sources:
  - url: https://github.com/owncloud/docs.git
    branches: 
    - master
  - url: https://github.com/owncloud/android.git
...
```

## Why Should This Change Be Made?

`make html` must generate on local machines a html site for previewing with local content, but not the content from the repository.

This is really a bad situation, because nothing can be tested anymore... :worried:

@settermjd @tboerger

**Comment by settermjd** [comment]
This is being fixed in #491.

**Comment by settermjd** [comment]
This was fixed in https://github.com/owncloud/docs/pull/475 with the addition of the `html-local` Makefile target.
